apps/datahub-e2e/src/e2e/search.cy.ts

```diff
@@ -301,10 +301,74 @@ describe('search', () => {
             'representationType',
             'revisionYear',
             'categoryKeyword',
+            'territories'
           ])
       })
+
+      describe('Search filters and query params from url', () => {
+        describe('When searching for a category in the url', () => {
+          beforeEach(() => {
+            cy.visit('/search?categoryKeyword=https:%2F%2Fdata.lillemetropole.fr%2Fthematique%2Fcategories%2Fadministration_action_publique')
+          })
+  
+          it('should not display the pagination', () => {
+            cy.get('mel-datahub-pagination-buttons').should('not.exist')
+          })
+        })
+      })
     })
   })
+
+  describe.only('Search filters and query params from url', () => {
```

Comment:
```suggestion
  describe('Search filters and query params from url', () => {
```

Reply:
FYI there is another `.only` preventing the other tests to be ran https://github.com/camptocamp/mel-dataplatform/pull/118/files#diff-1bc493d49d6a06a1b69de27f7d54204d6cd59b0bded1eb4f165b5872f409b4b4L273 I thought it was added on purpose.